src/App.tsx

```diff
@@ -10,33 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const onTabSelected = ({ id }: Tab) => {
```

Comment:
```suggestion
  const handleTabSelected = ({ id }: Tab) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364